lib/ReactViews/Workbench/Controls/ColorScaleRangeSection.jsx

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+import React from 'react';
+import ObserveModelMixin from '../../ObserveModelMixin';
+
+import Styles from './colorscalerange-section.scss';
+
+const ColorScaleRangeSection = React.createClass({
+    mixins: [ObserveModelMixin],
+
+    propTypes: {
+        item: React.PropTypes.object.isRequired
+    },
+
+    minRange: -50,
+    maxRange: 50,
+
+    updateRange() {
+        this.props.item.parameters.colorscalerange = [this.minRange, this.maxRange].join();
+        this.props.item.refresh();
+    },
+
+    changeRangeMin(event) {
+        this.minRange = event.target.value;
+    },
+
+    changeRangeMax(event) {
+        this.maxRange = event.target.value;
+    },
+
+    render() {
+        const item = this.props.item;
+        this.minRange = item.parameters.colorscalerange.split(',')[0];
+        this.maxRange = item.parameters.colorscalerange.split(',')[1];
+        if (!item.isNcWMS) {
+            return null;
+        }
+        return (
+            <div className={Styles.colorscalerange}>
+                <label>Colorscalerange: </label>
+                <label htmlFor="rangeMin">Min: </label>
+                <input type='text' name='rangeMin' defaultValue={this.minRange} onChange={this.changeRangeMin} />
+                <label htmlFor="rangeMax">Max: </label>
+                <input type='text' name='rangeMax' defaultValue={this.maxRange} onChange={this.changeRangeMax} />
+                <button type='button' onClick={this.updateRange} title="Update Range" className={Styles.btn}>Update Range</button>
```

Comment:
Put this in a form with an `onSubmit` so that the user can update the range by pressing enter.  I think that may fix the styling too.  If not, define `btn` in `colorscalerange-section.scss` to compose the one in `_buttons.scs`.